guides/common/modules/proc_installing-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment"]
+= Installing the {Team} Insights On-Premises advisor engine 
+
+You can pull the {Team} Insights On-Premises (IoP) advisor engine from the {Team} container registry if your {ProjectServer} has access to the registry.  
```

Comment:
Same here... For Foreman I think this should just read "The advisor engine" and for Satellite "The Insights in Satellite advisor engine"